src/main/java/games/strategy/triplea/delegate/InitializationDelegate.java

```diff
@@ -80,23 +80,17 @@ public boolean delegateCurrentlyRequiresUserInput() {
     return false;
   }
 
-  protected void init(final IDelegateBridge aBridge) {
-    initDestroyerArtillery(aBridge);
-    initShipyards(aBridge);
-    initTwoHitBattleship(aBridge);
-    initOriginalOwner(aBridge);
-    initTech(aBridge);
-    initSkipUnusedBids(aBridge.getData());
-    initAiStartingBonusIncome(aBridge);
-    initDeleteAssetsOfDisabledPlayers(aBridge);
-    initTransportedLandUnits(aBridge);
-    resetUnitState(aBridge);
-  }
+  protected void init(final IDelegateBridge delegateBridge) {
+    initDestroyerArtillery(delegateBridge);
+    initShipyards(delegateBridge);
+    initTwoHitBattleship(delegateBridge);
+    initOriginalOwner(delegateBridge);
+    initTech(delegateBridge);
+    initSkipUnusedBids(delegateBridge.getData());
+    initAiStartingBonusIncome(delegateBridge);
+    initDeleteAssetsOfDisabledPlayers(delegateBridge);
+    initTransportedLandUnits(delegateBridge);
 
-  /**
-   * The initTransportedLandUnits has some side effects, and we need to reset unit state to get rid of them.
-   */
-  private void resetUnitState(final IDelegateBridge aBridge) {
```

Comment:
Doesn't keeping this method make the high level init() method easier to read and more consistent in scope? It also helps to make clear what the lines in this method are doing.